adala/runtimes/_litellm.py

```diff
@@ -153,20 +153,32 @@ def _get_usage_dict(usage: Usage, model: str) -> Dict:
     return data
 
 
-class InstructorClientMixin:
+def resolve_litellm_model_and_provider(model_name: str, provider: str):
+    """
+    When using litellm.get_model_info() some models are accessed with their provider prefix
+    while others are not.
+
+    This helper function contains logic which resolves this for supported providers
+    """
+    if "/" in model_name:
+        model_name = "/".join(model_name.split("/")[1:])
+    provider = provider.lower()
+    if provider == "vertexai":
```

Comment:
why do we need to handle this case? Don't we control what's used as the provider? So we could just use "vertex_ai" instead of "vertexai", yeah?

Reply:
yes

Reply:
comes from here, feels wrong formatting it differently than the others 
https://github.com/HumanSignal/label-studio/pull/6865/files#diff-ea4ce962841ad8637bb0e2aecc79ea8ba18ae29b512a2f0841014169aac058afR14

Id rather keep this admittedly stupid logic in this one function

Reply:
Not the hill I'll die on, but seems like it'd be nice to have our names just match litellm in general since the name is somewhat arbitrary. If it bugs you though, I really don't mind this logic that much.